install/jms/docs/JMS3.1-ReleaseNotes.html

```diff
@@ -67,22 +65,20 @@ <h2>Jakarta EE Messaging TCK Facts</h2>
     </blockquote>
     <hr>
     <h2><a name="platform">Platform Notes</a></h2>
-    <p>The Jakarta EE Messaging TCK tests have been built with JDK 8 (1.8) and
-      tested with Java<font size="-2"><sup>TM</sup></font> Platform, Standard
-      Edition 8 (1.8) and 11.</p>
+    <p>The Jakarta EE Messaging TCK tests have been built with JDK 11 and tested
+      with Java<font size="-2"><sup>TM</sup></font> Platform, Standard Edition
+      11 and 17.</p>
     <p>The Jakarta EE Messaging TCK tests have been run on the following
       platforms:</p>
     <ul>
-      <li>CentOS Linux 7</li>
-      <li>Alpine Linux v3.12</li>
+      <li>TBD: include OS name + version</li>
```

Comment:
Shall we continue with CentOS Linux 7 and Alphine Linux V3.12?

Reply:
I'm not sure of which operation systems will be used by implementations to test with.  As a start, we can check which operating system is used for the [Eclipse CI](https://ci.eclipse.org/jakartaee-tck/job/checkOS/) which appears to currently be using `Debian GNU/Linux 10`.

Reply:
+1  for "Debian Linux 10" as in https://ci.eclipse.org/jakartaee-tck/job/checkOS/7/console

Reply:
Updated to `Debian Linux 10`